scripts/lib/callVibration.ts

```diff
@@ -0,0 +1,24 @@
+import System from '@smartface/native/device/system';
+import Invocation from '@smartface/native/util/iOS/invocation';
+
+export const callVibrate = (millisecond: number) => {
+    if (System.OS === System.OSType.ANDROID) {
+        System.vibrate({ millisecond: millisecond });
+    } else if (System.OS === System.OSType.IOS) {
+        let feedbackAlloc = Invocation.invokeClassMethod('UIImpactFeedbackGenerator', 'alloc', [], 'id');
+        // 0: Light , 1: Medium , 2: Heavy
+        //@ts-ignore
+        let argStyle = new Invocation.Argument({
```

Comment:
We should use const if the variable will not change :)

Reply:
DONE.